x-template.js

```diff
@@ -1,4 +1,5 @@
 /** Forgiving HTML parser which leverages innerHTML. */
+/*
```

Comment:
TODO: re-home `Forgiving` into demo or something for posterity (e.g. not shipped as a supported lib)

Reply:
Yep already planned — it’s in a comment here so I don’t need to keep resolving difficult git conflicts since we keep pulling functionality and tests forward from this change set and then rebasing this branch back on `main`.